### ISSUE TITLE
Can't start webserver of Zammad on Fedora 25

### ISSUE THREAD

**Issue by rbukovansky** [zammad/zammad#835]
### Infos:

* Used Zammad version: 1.3.0
* Operating system: Fedora 25
* Browser + version: Chrome 56

### Expected behavior:
`$ rails s -p 3000` starts server

### Actual behavior:
`$ rails s -p 3000` and I'm getting:

```
Usage:
  rails new APP_PATH [options]

Options:
  -r, [--ruby=PATH]                                      # Path to the Ruby binary of your choice
                                                         # Default: /usr/bin/ruby
  -m, [--template=TEMPLATE]                              # Path to some application template (can be a filesystem path or URL)
      [--skip-gemfile], [--no-skip-gemfile]              # Don't create a Gemfile
  -B, [--skip-bundle], [--no-skip-bundle]                # Don't run bundle install
  -G, [--skip-git], [--no-skip-git]                      # Skip .gitignore file
      [--skip-keeps], [--no-skip-keeps]                  # Skip source control .keep files
  -O, [--skip-active-record], [--no-skip-active-record]  # Skip Active Record files
  -S, [--skip-sprockets], [--no-skip-sprockets]          # Skip Sprockets files
      [--skip-spring], [--no-skip-spring]                # Don't install Spring application preloader
  -d, [--database=DATABASE]                              # Preconfigure for selected database (options: mysql/oracle/postgresql/sqlite3/frontbase/ibm_db/sqlserver/jdbcmysql/jdbcsqlite3/jdbcpostgresql/jdbc)
                                                         # Default: sqlite3
  -j, [--javascript=JAVASCRIPT]                          # Preconfigure for selected JavaScript library
                                                         # Default: jquery
  -J, [--skip-javascript], [--no-skip-javascript]        # Skip JavaScript files
      [--dev], [--no-dev]                                # Setup the application with Gemfile pointing to your Rails checkout
      [--edge], [--no-edge]                              # Setup the application with Gemfile pointing to Rails repository
      [--skip-turbolinks], [--no-skip-turbolinks]        # Skip turbolinks gem
  -T, [--skip-test-unit], [--no-skip-test-unit]          # Skip Test::Unit files
      [--rc=RC]                                          # Path to file containing extra configuration options for rails command
      [--no-rc], [--no-no-rc]                            # Skip loading of extra configuration options from .railsrc file

Runtime options:
  -f, [--force]                    # Overwrite files that already exist
  -p, [--pretend], [--no-pretend]  # Run but do not make any changes
  -q, [--quiet], [--no-quiet]      # Suppress status output
  -s, [--skip], [--no-skip]        # Skip files that already exist

Rails options:
  -h, [--help], [--no-help]        # Show this help message and quit
  -v, [--version], [--no-version]  # Show Rails version number and quit

Description:
    The 'rails new' command creates a new Rails application with a default
    directory structure and configuration at the path you specify.

    You can specify extra command-line arguments to be used every time
    'rails new' runs in the .railsrc configuration file in your home directory.

    Note that the arguments specified in the .railsrc file don't affect the
    defaults values shown above in this help message.

Example:
    rails new ~/Code/Ruby/weblog

    This generates a skeletal Rails installation in ~/Code/Ruby/weblog.
    See the README in the newly created application to get going.

```

Fedora 25
Ruby 2.3.3
Rails 4.2.7.1

**Comment by martini** [comment]
What is your current directory (pwd)?

It seems you are not in the Zammad root directory.

If you have Zammad installed via OS packages, just enter: `zammad run rails s -p 3000`

If you have Zammad installed via source, make sure that you are in the Zammad directory.

**Comment by rbukovansky** [comment]
1. Installed from source
2. pwd:
`/opt/zammad`

**Comment by martini** [comment]
It seems that you ruby/rails env is not set up correctly. I only can reproduce your error message if I try it outside the Zammad directory. So it's not Zammad related. It's ruby/rails related.


```
zammad me$ rails s -p 3000
=> Booting Puma
=> Rails 4.2.7.1 application starting in development on http://localhost:3000
...
```

```
tmp me$ rails s -p 3000
Usage:
  rails new APP_PATH [options]

Options:
  -r, [--ruby=PATH]                                      # Path to the Ruby binary of your choice
                                                         # Default: /Users/me/.rvm/rubies/ruby-2.3.1/bin/ruby
  -m, [--template=TEMPLATE]                              # Path to some application template (can be a filesystem path or URL)
  -d, [--database=DATABASE]                              # Preconfigure for selected database (options: mysql/oracle/postgresql/sqlite3/frontbase/ibm_db/sqlserver/jdbcmysql/jdbcsqlite3/jdbcpostgresql/jdbc)
                                                         # Default: sqlite3
  -j, [--javascript=JAVASCRIPT]                          # Preconfigure for selected JavaScript library
                                                         # Default: jquery
      [--skip-gemfile], [--no-skip-gemfile]              # Don't create a Gemfile
  -B, [--skip-bundle], [--no-skip-bundle]                # Don't run bundle install
  -G, [--skip-git], [--no-skip-git]                      # Skip .gitignore file
      [--skip-keeps], [--no-skip-keeps]                  # Skip source control .keep files
  -M, [--skip-action-mailer], [--no-skip-action-mailer]  # Skip Action Mailer files
  -O, [--skip-active-record], [--no-skip-active-record]  # Skip Active Record files
  -P, [--skip-puma], [--no-skip-puma]                    # Skip Puma related files
  -C, [--skip-action-cable], [--no-skip-action-cable]    # Skip Action Cable files
  -S, [--skip-sprockets], [--no-skip-sprockets]          # Skip Sprockets files
      [--skip-spring], [--no-skip-spring]                # Don't install Spring application preloader
      [--skip-listen], [--no-skip-listen]                # Don't generate configuration that depends on the listen gem
  -J, [--skip-javascript], [--no-skip-javascript]        # Skip JavaScript files
      [--skip-turbolinks], [--no-skip-turbolinks]        # Skip turbolinks gem
  -T, [--skip-test], [--no-skip-test]                    # Skip test files
      [--dev], [--no-dev]                                # Setup the application with Gemfile pointing to your Rails checkout
      [--edge], [--no-edge]                              # Setup the application with Gemfile pointing to Rails repository
      [--rc=RC]                                          # Path to file containing extra configuration options for rails command
      [--no-rc], [--no-no-rc]                            # Skip loading of extra configuration options from .railsrc file
      [--api], [--no-api]                                # Preconfigure smaller stack for API only apps

Runtime options:
  -f, [--force]                    # Overwrite files that already exist
  -p, [--pretend], [--no-pretend]  # Run but do not make any changes
  -q, [--quiet], [--no-quiet]      # Suppress status output
  -s, [--skip], [--no-skip]        # Skip files that already exist

Rails options:
  -h, [--help], [--no-help]        # Show this help message and quit
  -v, [--version], [--no-version]  # Show Rails version number and quit

Description:
    The 'rails new' command creates a new Rails application with a default
    directory structure and configuration at the path you specify.

    You can specify extra command-line arguments to be used every time
    'rails new' runs in the .railsrc configuration file in your home directory.

    Note that the arguments specified in the .railsrc file don't affect the
    defaults values shown above in this help message.

Example:
    rails new ~/Code/Ruby/weblog

    This generates a skeletal Rails installation in ~/Code/Ruby/weblog.
```

**Comment by rbukovansky** [comment]
Ok. I'm not familiar with ruby/rails env (I'm not dev, just sysop), trying any ruby/rails app for the first time.
Could you please let me know what check? Thanks a lot.

**Comment by martini** [comment]
Thanks for feedback. In general I would recommend to install the RPM/DEB package. It's quite easier because the whole ruby env is coming and will be updated automatically with the package.

-=> So you have not to care about this kind of issue. Also upgrades are done automatically. It's just fast and easy like you want it.

Anyway. If you install via source, I would recommend RVM (http://rvm.io). Just install RVM and Zammad with an dedicated user and all should work fine.

**Comment by rbukovansky** [comment]
Well, installation using RPM package doesn't work for me, because it want's to install nginx a postgresql, and I'm familiar more with LAMP stack.

I have installed rvm, but still not good (same error).
Just to be clear, I have followed this page: https://docs.zammad.org/en/latest/install-source.html when I was installing zammad.

**Comment by martini** [comment]
> Just to be clear, I have followed this page: https://docs.zammad.org/en/latest/install-source.html when I was installing zammad.

Yes. That's the correct way.

> Well, installation using RPM package doesn't work for me, because it want's to install nginx a postgresql, and I'm familiar more with LAMP stack.

One note. I used mysql for over 15 years as my favourite database in really large environments. I stumble through some issues with mysql and needed to look for alternatives. I tried postgresql, from this time I would never miss this experience. For all of my environments I moved to postgresql because it's just so easy, it just works, it just has **no issue** with 4 bit utf8 and other stuff. I started to ❤️ postgresql.

Same for nginx. It's just a so cool pease of software.

Just my 2 cents.
